front/src/components/list-item/item.tsx

```diff
@@ -24,6 +24,7 @@ import {
 	ListItemText,
 	ListItem as MUIListItem,
 	Skeleton,
+	Stack,
```

Comment:
I think you can revert all the changes made to this file, as we agreed that icons shouldn't appear here